### PR TITLE
Fix proxy to bypass CORS issue

### PR DESCRIPTION
**Pull request by skuridin**
Hello 🙂  This PR fixes the proxy problem, so you can continue working. 

What do you think about finishing the project?

### CODE REVIEW

File: app/services/linkService.js [hunk 1]
```diff
@@ -6,10 +6,6 @@ const POST_URL = `${BASE_URL}/shorten`;
 const requestCreateShortenLink = (url) => {
   fetch(POST_URL, {
     method: 'POST',
-    headers: {
-      'Content-Type': 'application/json',
```
Comment (skuridin) [inline app/services/linkService.js:10]: For some reasons `whatwg-fetch` uses `OPTIONS` instead of `POST` when you have content-type header.

Comment (JeanEspindola) [reply app/services/linkService.js:10]: Thanks Evgeniy.

I will resume the project and will let you know when is done.

**Comment by JeanEspindola** [comment]
Thanks Evgeniy, it's working fine the server now.

I couldn't test during the day at work, there due to the proxy, I face some
errors with DNS, but at home, everything works fine.

I'm now able to generate the shortcode correctly. I have to fix and
understand a little bit about the GET endpoints now. The stats is not
working fine in the application but if  I copy and paste in the browsers
tab, then it works. And for the get, just need to know how to get the url
from the location value, it's messing with the url itself.

Should the generated link work with no issues by being concatenated like
https://impraise-shorty.herokuapp.com/*****?

I will now keep working here to complete the project.

Thanks,
Jean

2017-12-06 13:29 GMT-02:00 Evgeniy Skuridin <notifications@github.com>:

**Comment by skuridin** [comment]
> Should the generated link work with no issues by being concatenated like
> https://impraise-shorty.herokuapp.com/*****?

Yep! [See here](https://gist.github.com/gabrielecirulli/c48c40fc45bbe49da774#get-shortcode).

Regarding problems with proxy, I believe you can use [proxy setting](https://webpack.js.org/configuration/dev-server/#devserver-proxy) in webpack-dev-server, since you are already using it.

**Comment by JeanEspindola** [comment]
Hi there.

Just to let you know, I completed the application last Saturday.

I will let Eva knows as well.

Thanks.

2017-12-07 11:13 GMT-02:00 Evgeniy Skuridin <notifications@github.com>:

**Pull request merged**